### PR TITLE
Formatting messages to be equal to CLI format

### PR DESCRIPTION
**Pull request by mlunoe**
This PR introduces a new format for log lines that follows the CLI (see here https://github.com/dcos/dcos-cli/pull/817/files#diff-8f3b06e62cf338c8e4e2ac6414447d26R260):

### CODE REVIEW

**Comment by MatApple** [comment]
@mlunoe legit failing tests

**Comment by mlunoe** [comment]
Oh man, different time on the CI than locally :/

File: src/js/stores/SystemLogStore.js [hunk 1]
```diff
@@ -117,13 +118,52 @@ class SystemLogStore extends BaseStore {
...
+      const {fields = {}} = entry;
+      let lineData = [];
+      // entry.realtime_timestamp returns a unix time in microseconds
+      // https://www.freedesktop.org/software/systemd/man/sd_journal_get_realtime_usec.html
+      if (typeof entry.realtime_timestamp === 'number') {
+        lineData.push(msToCTime(entry.realtime_timestamp/1000));
+      }
+
+      // Optional fields
+      ['_HOSTNAME', 'SYSLOG_IDENTIFIER'].forEach((optionalField) => {
```
Comment (MatApple) [inline src/js/stores/SystemLogStore.js]: Let's put these these in a constants file since we reuse them.

Comment (mlunoe) [inline src/js/stores/SystemLogStore.js]: Ok, moved them to a constants file

**Comment by jfurrow** [comment]
To me this seems like this makes the logs really hard to parse because the length of the prefix is so long. Maybe the prefix could be on a new line of its own, or could be a lighter shade of grey to offset it from the output? Just an idea, maybe this isn't a problem!

**Comment by mlunoe** [comment]
I cannot change the format. It has to follow the CLI exactly, as it does now

**Comment by jfurrow** [comment]
@mlunoe Okay cool, thanks!

**Comment by jfurrow** [comment]
Works great, thanks for adding tests!
